docs/server-side/v2/how--to-guides/nodejs-server-side-sdk.mdx

```diff
@@ -0,0 +1,226 @@
+---
+title: Server side Node.js SDK
+nav: 3
+---
+
+The 100ms Node.js SDK provides an easy-to-use wrapper around [100ms REST APIs](https://www.100ms.live/docs/server-side/v2/introduction/request-and-response) to be used on the server side.
+
+🔧 **This is beta, APIs might change**
+
+Some examples on how to use the SDK can be found in the `examples/nodejs` folder on [the source code](https://github.com/100mslive/server-sdks).
+
+## Installation
+
+Install the package with:
+
+```
+# via NPM:
+npm install --save @100mslive/server-sdk
+
+# via Yarn:
+yarn add @100mslive/server-sdk
+```
+
+## Usage
+
+The SDK needs to be configured with the **Access Key** and **App Secret** from the [100ms Dashboard's Developer Section](https://dashboard.100ms.live/developer). This can be done in 2 ways:
+
+1. Passing in the credentials when initializing the SDK.
+
+```js
+import HMS from "@100mslive/server-sdk";
+
+const hms = new HMS.SDK(accessKey, secret);
+```
+
+**OR**
+
+2. Configuring Environment variables with the credentials,
+
+```
+HMS_ACCESS_KEY=accessKey123 // access key
+HMS_SECRET=secret456 // app secret
+```
+
+Then initializing the SDK like this:
+
+```js
+import HMS from "@100mslive/server-sdk";
+
+const hms = new HMS.SDK();
+```
+
+### Generate Management token for server side APIs
+
+```js
+import HMS from "@100mslive/server-sdk";
+
+const hms = new HMS.SDK();
+console.log(await hms.auth.getManagementToken());
+// with token options -
+const tokenConfig = {
+  issuedAt,
+  notValidBefore,
+  validForSeconds,
+};
+console.log(await hms.auth.getManagementToken(tokenConfig));
+```
+
+### Generating Auth token for client SDKs
+
+```js
+import HMS from "@100mslive/server-sdk";
+
+const hms = new HMS.SDK();
+const tokenConfig = { roomId, role, userId };
+console.log(await hms.auth.getAuthToken());
+// with additional token options -
+const additionalTokenConfig = {
+  roomId,
+  role,
+  userId,
+  issuedAt,
+  notValidBefore,
+  validForSeconds,
+};
+console.log(await hms.auth.getAuthToken(additionalTokenConfig));
+```
+
+### Creating and updating room
+
+```js
+import HMS from "@100mslive/server-sdk";
+
+const hms = new HMS.SDK();
+// creating a room -
+const room = await hms.rooms.create();
+// with room options -
+const roomCreateOptions = {
+  name,
+  description,
+  template_id,
+  recording_info,
+  region,
+};
+const roomWithOptions = await hms.rooms.create(roomCreateOptions);
+
+// updating a room -
+const roomUpdateOptions = { name };
+const updatedRoom = await hms.rooms.update(room.id, roomUpdateOptions);
+console.log(room, roomWithOptions, updatedRoom);
+```
+
+### > Usage with TypeScript
```

Comment:
the `>` be removed? 

also this is in the middle of usage sections around 100ms entities (rooms, room codes). typescript config can be in the intro?